core/frontend/src/quantity-formatting/QuantityFormatter.ts

```diff
@@ -22,7 +23,7 @@ import { BasicUnitsProvider } from "./BasicUnitsProvider";
  * "imperial" -> PresentationUnitSystem.BritishImperial ("british-imperial")
  * "usCustomary" -> PresentationUnitSystem.UsCustomary ("us-customary")
  * "usSurvey" -> PresentationUnitSystem.UsSurvey ("us-survey")
- * @alpha
+ * @beta
  */
 export type UnitSystemKey = "metric" | "imperial" | "usCustomary" | "usSurvey";
```

Comment:
Before we make this beta, do we want to align our naming with Presentation Unit Systems?

Reply:
So are you suggesting to use the name "PresentationUnitSystemKey", change the entries to exactly match the strings in PresentationUnitSystem enum or both?  I avoided adding a dependency on the Presentation package and using the enum directly to avoid a circular dependency but I could use the exact same strings, it would just be a breaking change... but I guess that is OK since it was alpha.

Reply:
Yes, I was thinking before it moves to beta this is probably the last chance to align the two better. Don't have a good suggestion at the moment, I'll review more closely tomorrow (have to review @ColinKerr's #902 as well).

It definitely doesn't make sense to add a presentation dependency. Maybe at the very least have the string values align even if it's not the same type.

Looping @grigasp in as well.

Reply:
I think it's inevitable for presentation packages to depend on this package in the future, so definitely not worth adding a presentation-common dependency in here. Having the string values match would be helpful until presentation has that dependency, but afterwards I think we're going to deprecate `PresentationUnitSystem` and switch to `UnitSystemKey` or at least have some function that maps from one to the other.

Also I should mention that `PresentationUnitSystem` is not a really good name for what it stands for. IMO I should've named it `PresentationUnitSystemGroup`, because it stands for a group of unit systems.

Reply:
- My vote would be to not change the string values defined by UnitSystemKey, but possibly change the strings used in PresentationUnitSystem enum to match, especially since this would not cause any side effects for existing API consumers of either package. I remember looking at changing QuantityFormatter to use the strings from the PresentationUnitSystem enum once, and it caused other issues with the JSON that was being serialized because I had used the UnitSystemKey as the key in the persisted JSON object and JSON did not like a "-" in the name of a key.
- The name "Presentation Unit System" is really only exposed as a string in the UI because we needed a name the would not be confused with the "UnitSystem"  from the units schema, for the few people that actually look at the units schema.  I can change the default label to "Unit System Group" to be more accurate but exiting apps like it1 and it3 already use the label "Unit System" in their UI.  I thought adding "Presentation" to it would be less confusing for existing UI's to adopt.

Reply:
@grigasp if you are not opposed I will make the following change.
```
/** @alpha */
export enum PresentationUnitSystem {
  Metric = "metric",
  BritishImperial = "imperial",
  UsCustomary = "usCustomary",
  UsSurvey = "usSurvey",
}
```

Reply:
No can do - these strings are sent to the backend and used by the addon. So the addon needs to be updated to handle both variants until we get a chance to bump RPC major version.

If you don't want to change `UnitSystemKey`, I'd say just leave them as is for now. I see `presentation-common` already has a dependency on `imodeljs-quantity` - for now I'll add mapping functions to map between these two types and will switch to using `UnitSystemKey` when the time is right.

Reply:
Sounds good, I'll resolve this for now and we can circle back to it separately.